CollectionOfHelpers/CollectionOfHelpersTests/DictionaryExtensionTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using NUnit.Framework;
+using CollectionOfHelpers.GeneralExtensions;
+
+namespace CollectionOfHelpersTests
+{
+    /// <summary>
+    /// Summary description for DictionaryExtensions
+    /// </summary>
+    [TestFixture]
+    public class DictionaryExtensionTests
+    {
+        [Test]
+        public void Dictionary_tryadd_keyvalueexists()
+        {
+            //Arrange
+            var dic = new Dictionary<string, string>();
+            dic.Add("Hi", "there");
+
+            //act
+            var actual = dic.TryAdd("Hi", "there");
+
+            //assert
+            Assert.IsFalse(actual);
+            Assert.AreEqual(dic.Count, 1);
```

Comment:
should be AreEqual(expected, actual) - otherwise test failure messages can be misleading